# HG changeset patch
# User Stefan Mahr <dac922@gmx.de>
# Date 1374325610 14400
#      Sat Jul 20 09:06:50 2013 -0400
# Node ID ddc62f0c7a35a1d21c501827e0616c3c1cc626db
# Parent  154899fa42e3f8de1af3a269059a177ecb6886ef
Fix mistake from changeset 942d892524b3.

script/plot/subplot.m: Using "hsubplot" in place of "tmp".

diff --git a/scripts/plot/subplot.m b/scripts/plot/subplot.m
--- a/scripts/plot/subplot.m
+++ b/scripts/plot/subplot.m
@@ -228,17 +228,17 @@ function h = subplot (varargin)
           if (! (x0 >= objx1 || x1 <= objx0 || y0 >= objy1 || y1 <= objy0))
             delete (child);
           endif
         endif
       endif
     endfor
 
     if (found)
-      set (cf, "currentaxes", tmp);
+      set (cf, "currentaxes", hsubplot);
     else
       hsubplot = axes ("box", "off",
                        "position", pos,
                        "looseinset", looseinset,
                        "activepositionproperty", activepositionproperty,
                        varargin{:});
       addproperty ("subplot_align", hsubplot, "boolean", true);
       addlistener (hsubplot, "position", @subplot_align);
